app/controllers/profiles_controller.rb

```diff
@@ -8,10 +8,7 @@ def index
   end
 
   def show
-    @solutions = @user.solutions.published.
-      order(num_stars: :desc, updated_at: :desc).
-      includes(:exercise, :track).
-      first(3)
+    @solutions = Solution::SearchUserSolutions.(@user, status: :published, per: 3)
```

Comment:
This is much faster (like 1s -> 0.02sec) so we should check if there are other places that could use this too.

Reply:
Is that due to the fact that the sorting is now changing to just sorting by `num_stars` and no longer using `updated_at`?

Reply:
It's that it uses cloudsearch not Sql